README.md

```diff
@@ -76,6 +54,50 @@ To run the CLI, install `@azure/static-web-apps-cli` and the [Azure Functions Co
 
 ## Options
 
+### apiDir
+
+The directory where the `render` Azure function for SSR will be placed. Most of the time, you shouldn't need to set this.
+
+By default, the adapter will output the `render` Azure function for SSR in the `build/server` folder. If you want to output it to a different directory instead (e.g. if you have additional Azure functions to deploy), you can set this option.
+
+```js
+import azure from 'svelte-adapter-azure-swa';
+
+export default {
+	kit: {
+		...
+		adapter: azure({
+			apiDir: 'custom/api'
```

Comment:
Right now this is the folder the function folder will be placed in (so the function will be output in `custom/api/render/index.js`). But this could potentially conflict with another function named "render" in that folder. Should this be the path to the render function instead (so the example above would be custom/api/render)? Not sure which is more intuitive.

Reply:
The docs could use more clarity on this either way.

Reply:
Though I guess if we ever implement #25 there could be multiple azure functions, so we likely want the API directory instead of the individual function directory. Probably at least mention that the `render` folder is reserved (or provide a way to customize it)

Reply:
Does `apiDir` even need to be customizable? Currently, the contents of `apiDir` are emptied by the `adapt(builder)` function when it runs `builder.rimraf(apiDir)`, so it seems like something that should just be inside a temporary build directory. Unless I'm missing something, if anyone needs to create a custom Azure function that coexists with SvelteKit they can't do it now (because of `rimraf`) and won't need to later (because of #25).

Reply:
Good catch on `builder.rimraf(apiDir)` - that was a miss on my part. That shouldn't be run if a custom apiDir is provided.

Today it _is_ possible to have other Azure functions in the `api` directory. For example, this would expose a separate API function outside of the SvelteKit app available at `/api/random`:

```
api/
├── render # generated by adapter
└── random
```

Moving the server render function inside `build` with this PR means you can't do that anymore. By exposing `apiDir`, this allows the user to output the render function in a folder with other API functions they want to deploy.

I do think it's a niche use case since most of the time you'll just put the logic in your SvelteKit app, but #42 indicates that some people want it.

I don't think #25 makes it unnecessary - that's more about splitting the SvelteKit app into multiple Azure functions, so it wouldn't help if you have an Azure function separate from SvelteKit.

Reply:
I see. Is there another way to resolve the problem described by #42? They didn't explain why they wanted to output to the same directory. Perhaps it would be better to let them specify a directory of additional functions that the adapter copies into the temporary build directory at build time. If that approach was used, conflicts could be avoided automatically by simply renaming the render function(s) internally when a conflicting filename is found in the provided directory. Otherwise, it will be difficult to reliably remove stale files from previous builds without accidentally deleting important user files. The problem might not be that big of a deal right now, but it will be when multiple render functions are generated with #25. There would be no guaranteed way to determine if a function is stale from a previous build or if it is user-provided.

Reply:
I don't love copying their functions into the build directory because then import paths could get messed up - it feels too unexpected.

I think the main concern is conflicts, i.e. having our generated function folders collide with the folder names they choose for their own functions. A good solution to that would be to reserve a prefix (e.g. sveltekit_*) that we prefix all our functions folders with (only 1 folder until #25). Then we could reliably remove stale files by only rimraf-ing folders with that prefix.

I can't think of any obvious problems with that, and it should make things future proof when #25 is implemented, since we can generate sveltekit_1, sveltekit_2, etc. folders.

Reply:
I thought the function's folder was supposed to be self-contained, but if that's not the case then using a reserved prefix should do the trick.

Reply:
The function is now output to a `sk_render` directory and the docs have been updated to call out that folders prefixed with `sk_render` are reserved, as are routes starting with __render.